### PR TITLE
Update TopBarGump.cs

### PR DESCRIPTION
**Pull request by tonyuo**
Changed Map, Paperdoll, Inventory, Journal, Chat and Help to take these names from Cliloc

### CODE REVIEW

File: src/Game/UI/Gumps/TopBarGump.cs [hunk 1]
```diff
@@ -109,9 +109,18 @@ private TopBarGump() : base(0, 0)
...
+                cliloc.GetString(3000133,ResGumps.Paperdoll), 
+                cliloc.GetString(3000431,ResGumps.Inventory), 
+                cliloc.GetString(3000129,ResGumps.Journal), 
+                cliloc.GetString(3000131,ResGumps.Chat), 
+                cliloc.GetString(3000134,ResGumps.Help),     
+                ResGumps.WorldMap), 
+                ResGumps.Info), 
+                ResGumps.Debug), 
+                ResGumps.NetStats), 
+                ResGumps.UOStore),
```
Comment (andreakarasho) [inline src/Game/UI/Gumps/TopBarGump.cs]: Can you add the translation for uostore & global chat too?

Here the clilocs thanks
cliloc.GetString(1158008, ResGumps.UOStore),
cliloc.GetString(1158390, ResGumps.GlobalChat)

**Comment by andreakarasho** [comment]
also missing the `cliloc` object declaration

File: src/Game/UI/Gumps/TopBarGump.cs (hunk 1 above)
Comment (tonyuo) [reply src/Game/UI/Gumps/TopBarGump.cs]: Sure, I didn't find them at Clilocs

**Comment by tonyuo** [comment]
I found a cliloc por World Map too 1015233

**Comment by andreakarasho** [comment]
you still need to add `var cliloc = ClilocLoader.Instance;`

**Comment by tonyuo** [comment]
Just something about cliloc.GetString(3000169,ResGumps.NetStats), 

It doesn't say NetStats, It cliloc is Connection (don't know if you agree with that)

File: src/Game/UI/Gumps/TopBarGump.cs [hunk 2]
```diff
@@ -107,11 +107,22 @@ private TopBarGump() : base(0, 0)
...
+                cliloc.GetString(3000430,ResGumps.Map), 
+                cliloc.GetString(3000133,ResGumps.Paperdoll), 
+                cliloc.GetString(3000431,ResGumps.Inventory), 
+                cliloc.GetString(3000129,ResGumps.Journal), 
+                cliloc.GetString(3000131,ResGumps.Chat), 
+                cliloc.GetString(3000134,ResGumps.Help),     
+                cliloc.GetString(1015233,ResGumps.WorldMap), 
+                cliloc.GetString(1079449,ResGumps.Info), 
+                cliloc.GetString(1042237,ResGumps.Debug), 
+                cliloc.GetString(3000169,ResGumps.NetStats),
```
Comment (andreakarasho) [inline src/Game/UI/Gumps/TopBarGump.cs]: add a whitespace between the clicloc and the comma please

Comment (tonyuo) [reply src/Game/UI/Gumps/TopBarGump.cs]: > add a whitespace between the clicloc and the comma please

space between cliloc and coma, or comma and ResGumps?

**Pull request merged**